src/driver/graphic.rs

```diff
@@ -780,8 +780,11 @@ impl Default for StencilMode {
     }
 }
 
+/// The state of the vertex input stage of the graphics pipeline.
 #[derive(Clone, Debug, Default)]
-pub(super) struct VertexInputState {
+pub struct VertexInputState {
```

Comment:
_Probably_ should add this the the main `lib.rs` prelude; it's an odd type but I got into the questionable habit of putting all public types in there.

Reply:
Totally fine by me. :+1: